server/portal/apps/workspace/api/views.py

```diff
@@ -145,30 +145,28 @@ def delete(self, request, *args, **kwargs):
         )
 
     def post(self, request, *args, **kwargs):
-        agave = request.user.tapis_oauth.client
+        tapis = request.user.tapis_oauth.client
         job_post = json.loads(request.body)
-        job_id = job_post.get('job_id')
+        job_uuid = job_post.get('job_uuid')
         job_action = job_post.get('action')
 
-        if job_id and job_action:
+        if job_uuid and job_action:
             # resubmit job
             if job_action == 'resubmit':
-                METRICS.info("user:{} is resubmitting job id:{}".format(request.user.username, job_id))
+                METRICS.info("user:{} is resubmitting job id:{}".format(request.user.username, job_uuid))
+                data = tapis.jobs.resubmitJob(jobUuid=job_uuid)
             # cancel job / stop job
             else:
-                METRICS.info("user:{} is canceling/stopping job id:{}".format(request.user.username, job_id))
-
-            data = agave.jobs.manage(jobId=job_id, body={"action": job_action})
+                METRICS.info("user:{} is canceling/stopping job id:{}".format(request.user.username, job_uuid))
+                data = tapis.jobs.cancelJob(jobUuid=job_uuid)
 
-            if job_action == 'resubmit':
-                if "id" in data:
-                    job = JobSubmission.objects.create(
-                        user=request.user,
-                        jobId=data["id"]
-                    )
-                    job.save()
```

Comment:
Are we ready to get rid of the JobSubmission model?

Reply:
> I've removed the use of JobSubmission when resubmitting a job assuming that the tag information (including portal information) is copied over in Tapis V3 when resubmitting a job.

Perhaps we should wait until the tag search is ready to remove use of the JobSubmission object

Reply:
I removed it because it was removed in the get request. But I'll add it back here just in case!